app/scripts/controllers/transactions/index.js

```diff
@@ -704,13 +744,15 @@ export default class TransactionController extends EventEmitter {
    * @param origin
    * @param transactionType
    * @param sendFlowHistory
+   * @param actionId
```

Comment:
Could we add some comments to indicate what is happening when an actionId is specified?

Reply:
yep done.